services/validators.js

```diff
@@ -17,3 +17,6 @@ export const optionalDottedVersionNClausesWithOptionalSuffix =
 // TODO This accepts URLs with query strings and fragments, which for some
 // purposes should be rejected.
 export const optionalUrl = Joi.string().uri({ scheme: ['http', 'https'] })
+export const fileSize = Joi.string()
+  .regex(/^[0-9]+(b|kb|mb|gb|tb)$/i)
+  .required()
```

Comment:
I guess we can't really consolidate this pattern with the one we already use for test validation against our badges that display file size, but perhaps worth something like an inline comment to indicate that this is really to reuse the pattern we use for the Shields server config as opposed to API response or badge content? I could easily picture myself reviewing a hypothetical future PR that adds a file size badge for some new service Foo, seeing `fileSize` used in the tests and thinking all was well.

https://github.com/badges/shields/blob/d2bde17fd6c1d50156e3c5f1f328940af420fc16/services/test-validators.js#L96-L98